# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/PublicKeyCredential.cpp
# Commit: 287bedb227e8
# Full Hash: 287bedb227e89933347633242efe1acd4236ad70
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-12 09:32:07
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r a01c165a2524 -r 287bedb227e8 dom/webauthn/PublicKeyCredential.cpp
--- a/dom/webauthn/PublicKeyCredential.cpp	Wed Feb 12 02:12:58 2025 +0000
+++ b/dom/webauthn/PublicKeyCredential.cpp	Wed Feb 12 02:12:58 2025 +0000
@@ -15,7 +15,7 @@
 #include "mozilla/dom/Promise.h"
 #include "mozilla/dom/PublicKeyCredential.h"
 #include "mozilla/dom/WebAuthenticationBinding.h"
-#include "mozilla/dom/WebAuthnManager.h"
+#include "mozilla/dom/WebAuthnHandler.h"
 #include "nsCycleCollectionParticipant.h"
 
 #ifdef MOZ_WIDGET_ANDROID
@@ -121,9 +121,9 @@
     return nullptr;
   }
 
-  RefPtr<WebAuthnManager> manager =
-      window->Navigator()->Credentials()->GetWebAuthnManager();
-  return manager->IsUVPAA(aGlobal, aError);
+  RefPtr<WebAuthnHandler> handler =
+      window->Navigator()->Credentials()->GetWebAuthnHandler();
+  return handler->IsUVPAA(aGlobal, aError);
 }
 
 /* static */